packages/lambda-functions/template.yaml

```diff
@@ -3,40 +3,29 @@ Transform: AWS::Serverless-2016-10-31
 Description: >
   lambda-functions
 
-# More info about Globals: https://github.com/awslabs/serverless-application-model/blob/master/docs/globals.rst
 Globals:
   Function:
-    Timeout: 120
+    Timeout: 360
```

Comment:
Does this actually work?

Reply:
Yep. This was necessary to prevent the lambda from timing out in local testing